card/mod/core/spec/set/all/permissions_spec.rb

```diff
@@ -99,7 +99,12 @@ def assert_not_locked card, msg=""
     it "reader setting" do
       Card.where(trash: false).each do |ca|
         rule_id, rule_class = ca.permission_rule_id_and_class(:read)
-        next if rule_class != ca.read_rule_class
+        if rule_id != ca.read_rule_id
+          puts "read rule id mismatch for #{ca.name} " \
+               "expected #{rule_class}/#{rule_id}, " \
+               "got #{ca.read_rule_class}/#{ca.read_rule_id}"
```

Comment:
This is already part of the expect statements. To change the failure message pass the string as second argument to `to`.